### ISSUE TITLE
Document how to use scroll events/state

### ISSUE THREAD

**Issue by rooby** [MurhafSousli/ngx-scrollbar#105]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ x ] feature request
- [ ] question
```

### Desired functionality
In the changelog for the 3.0.0 version it mentions:

> (scrollState) output is removed, to get the scroll event use the component ref, example:
> ```
> @ViewChild(NgScrollbar) ngScrollbar: NgScrollbar;
> ngScrollbar.scrollable.elementScrolled().subscribe(e => console.log(e))
> ```

However there isn't any mention of this in the readme.

It would be good to show a basic example of accessing scroll state.

Also, If I add that code above in my component, when it initialises it throws this error, even if it's in ngAfterViewInit:

> Cannot read property 'scrollable' of undefined

It does actually work afterwards though.

If the documentation shows code that avoids this issue or makes mention of how to avoid this issue that would be great.

**Comment by vasiltabakov** [comment]
@rooby for the ViewChild issue you can use something like:
```
@ViewChild('scrollBar') set scrollBar(scrollbar: NgScrollbar) {
     // subscribe
}
```

**Comment by MurhafSousli** [comment]
@rooby No, you should not expect that to work! in Typescript you must use `this` to get the element, 
Wait for the view to initialize and then `subscribe` on `elementScrolled()`

```ts
@ViewChild(NgScrollbar) scrollbarRef: NgScrollbar;

ngAfterViewInit() {
   this.scrollbarRef.scrollable.elementScrolled().subscribe(e => console.log(e))
}
```

**Comment by rooby** [comment]
@MurhafSousli 
I got that to work because I read the changelog. (my previous error randomly went away. I'm not sure why it was coming up before.)

My point is that this should be in the readme so that people know how to do it without having to search through issues or changelog. It seems strange that that this isn't in the main README docs.
